receiver/datadogreceiver/translator_test.go

```diff
@@ -67,18 +68,20 @@ var data = [2]interface{}{
 var output ptrace.Traces
 
 func TestTracePayloadV05Unmarshalling(t *testing.T) {
```

Comment:
Since this PR adds support for v0.7, could we also have tests covering traces sent for that version?

Reply:
Sorry this is sorta unclear,  The Marshalling and Unmarshalling of TracePayload is v0.7, it is incidentley done in other tests. can make an explicit one i suppose

Reply:
Tests added